human_aware_rl/imitation/behavior_cloning_tf2_test.py

```diff
@@ -121,8 +121,7 @@ def test_lstm_evaluation(self):
 
     def test_lstm_save_and_load(self):
         self.bc_params['use_lstm'] = True
-        model = build_bc_model(**self.bc_params)
-        save_bc_model(self.model_dir, model, self.bc_params)
+        model = train_bc_model(self.model_dir, self.bc_params)
```

Comment:
Same as the non-LSTM case, I would argue we should try and isolate our unit tests as much as possible